indico/util/benchmark.py

```diff
@@ -58,12 +58,12 @@ def __str__(self):
     def print_result(self, slow=float('inf'), veryslow=float('inf')):
         duration = float(self)
         if duration == float('-inf'):
-            print(cformat('%{blue!}skipped'))
+            click.secho('skipped', fg='blue')
         elif duration == float('inf'):
-            print(cformat('%{red}running'))
+            click.secho('running', fg='red')
         elif duration >= veryslow:
-            print(cformat('%{red!}{}').format(self))
+            click.secho(f'{self}', fg='red', bold=True)
```

Comment:
`click.secho(str(self), ...)` if you need an explicit conversion to str (same below)